index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Countdown Timer</title>
+</head>
+<body>
+    
+    <h1>New Year Eve</h1>
+		<div class="count-down-container">
+			 <ul id="days">
+				 <li class="days-count count">
+					 <!-- Count generated dynamically with JavaScript-->
+				 </li>
+				 <li>Days</li>
+			 </ul>
```

Comment:
I think it is better to use div tags instead of li tags because the content is not a list. So I think it is always good to choose the appropriate tag for content.